rd_ui/app/scripts/visualizations/map.js

```diff
@@ -12,7 +12,9 @@
     var editTemplate = '<map-editor></map-editor>';
     var defaultOptions = {
       'height': 500,
-      'draw': 'Marker'
+      'draw': 'Marker',
+      'draw_options' : ['Marker','Color'],
```

Comment:
It's worth moving `draw_options` to the `$scope` as well, as there is no point in saving it along with every map visualization.